largestNumber.js

```diff
@@ -0,0 +1,13 @@
+function largest(arr) {
+    return Math.max(...arr)
+}
+
+console.log(largest([3, 5, 1, 37, 26]))
```

Comment:
don't copy code if you don't understand it

Reply:
done